azure-mgmt-hdinsight/azure/mgmt/hdinsight/models/hd_insight_management_client_enums.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum
+
+
+class DirectoryType(Enum):
+
+    active_directory = "ActiveDirectory"
+
+
+class OSType(Enum):
+
+    windows = "Windows"
+    linux = "Linux"
+
+
+class Tier(Enum):
+
+    standard = "Standard"
+    premium = "Premium"
+
+
+class HDInsightClusterProvisioningState(Enum):
+
+    in_progress = "InProgress"
+    failed = "Failed"
+    succeeded = "Succeeded"
+    canceled = "Canceled"
+    deleting = "Deleting"
+
+
+class AsyncOperationState(Enum):
+
+    in_progress = "InProgress"
+    succeeded = "Succeeded"
+    failed = "Failed"
+
+
+class ApplicationName(Enum):
+
```

Comment:
This looks like a bug, either in code generation or the original spec.

Reply:
The pull request for this is here: https://github.com/Azure/azure-rest-api-specs/pull/1938

Reply:
@wawon-msft regenerated with the fix